src/System/executable_query.rs

```diff
@@ -24,6 +26,9 @@ impl QueryResult {
             QueryResult::Refinement(true) => satisfied(query_str),
             QueryResult::Refinement(false) => not_satisfied(query_str),
 
+            QueryResult::Reachability(true, _) => {println!("{} -- reachable", query_str)},
+            QueryResult::Reachability(false, _) => {println!("{} -- not reachable", query_str)},
+
```

Comment:
Should probably turn the rightsides here into functions like the others.
Also the success should print the path

Reply:
Fix